src/software/ai/hl/stp/play/stop_play.cpp

```diff
@@ -95,10 +93,12 @@ void StopPlay::getNextTactics(TacticCoroutine::push_type &yield,
 
         std::get<0>(crease_defender_tactics)
             ->updateControlParams(world_ptr->ball().position(),
-                                  TbotsProto::CreaseDefenderAlignment::LEFT, stop_mode);
+                                  TbotsProto::CreaseDefenderAlignment::LEFT, stop_mode,
+                                  TbotsProto::BallStealMode::IGNORE);
         std::get<1>(crease_defender_tactics)
             ->updateControlParams(world_ptr->ball().position(),
-                                  TbotsProto::CreaseDefenderAlignment::RIGHT, stop_mode);
+                                  TbotsProto::CreaseDefenderAlignment::RIGHT, stop_mode,
+                                  TbotsProto::BallStealMode::IGNORE);
```

Comment:
We do not want crease defenders to steal during stop play, thus new control param was added.